### PR TITLE
[Merged by Bors] - feat: monoid action on finprod

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

Is adding this import OK? If not then it seems to me that this lemma needs to go by itself in a new file.

### CODE REVIEW

File: Mathlib/GroupTheory/GroupAction/BigOperators.lean [hunk 1]
```diff
@@ -67,4 +68,9 @@ theorem Finset.smul_prod {r : α} {f : γ → β} {s : Finset γ} :
   map_prod (MulDistribMulAction.toMonoidHom β r) f s
 #align finset.smul_prod Finset.smul_prod
 
+theorem smul_finprod [Finite γ] {f : γ → β} (r : α) :
```
Comment (eric-wieser) [inline Mathlib/GroupTheory/GroupAction/BigOperators.lean:71]: I think you want to state this for `γ : Sort*`?

Comment (kbuzzard) [reply Mathlib/GroupTheory/GroupAction/BigOperators.lean:71]: ha ha that might be the case but then I can't prove it :-)

Comment (eric-wieser) [reply Mathlib/GroupTheory/GroupAction/BigOperators.lean:71]: I think you then have to use `cases nonempty_fintype (PLift γ)`, and you can probably work out the rest by looking for other uses of that pattern.

**Comment by eric-wieser** [comment]
bors merge

Thanks!

**Pull request closed without merging**